.travis.yml

```diff
@@ -10,14 +10,26 @@ d:
  - dmd-2.068.2
  - ldc-1.0.0-alpha1
  - ldc-0.17.0
+env:
+ - x86_64_BUILD=TRUE
+ - x86_32_BUILD=TRUE
+matrix:
+  exclude:
+    - osx
+    - x86_32_BUILD=TRUE
+addons:
+  apt:
+    packages:
+        gcc-multilib
```

Comment:
I like the way it is done now - it adds only a minimal overhead and we can exactly pick which versions we want to test under 32bit.